gadgets/traceloop/go/program.go

```diff
@@ -0,0 +1,788 @@
+// Copyright 2024 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"sort"
+	"strconv"
+	"sync"
+	"syscall"
+	"time"
+	"unsafe"
+
+	api "github.com/inspektor-gadget/inspektor-gadget/wasmapi/go"
+	tracelooptypes "github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/traceloop/types"
+	eventtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
```

Comment:
We should not import anything else than `wasmapi/go/` in a gadget.

* tracelooptypes is the builtin gadget. If you need something from there, it should be moved to this file directly.
* if you need something from eventtypes, it should be added to wasmapi/go/. But it seems the current usage should not be used like that: we can do without Time, and the other things belong to builtin gadgets.